config/crd/bases/integreatly.org_grafanadatasources.yaml

```diff
@@ -58,10 +58,14 @@ spec:
                         json options See https://grafana.com/docs/administration/provisioning/#datasources
                       properties:
                         addCorsHeader:
-                          description: ' Useful fields for clickhouse datasource  See
```

Comment:
Looks like the CI fails because of this. Should I revert this or do you want to merge like this?